src/sentry/api/serializers/models/release.py

```diff
@@ -255,24 +263,32 @@ def __get_release_data_with_environment(self, project, item_list, environment):
 
     def get_attrs(self, item_list, user, **kwargs):
         project = kwargs.get("project")
+
+        # Some code paths pass an environment object, other pass a list of
```

Comment:
would patch up those codepaths instead of doing this

Reply:
Yeah. I mostly didn't do it because I couldn't figure what all the code paths are :(